tests/cases/fourslash/formattingJsxTexts4.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+// Github issue #55293
+
+//@Filename: file.tsx
+//// function foo() {
+//// const a = <ns: foobar   x : test1   x :test2="string"  x:test3={true?1:0}  />;
+////
+//// return a;
+//// }
+
+format.document();
+
+verify.currentFileContentIs(
+`function foo() {
+    const a = <ns:foobar   x:test1   x:test2="string"  x:test3={true ? 1 : 0} />;

```

Comment:
Is it normal that these extra spaces are left inside this element? I would have guessed they'd be removed. Is there another rule that is missing the info about being able to remove spaces?